### PR TITLE
Changed BalanceRow to be a single Text component.

### LINKED ISSUE #3043: [BUG] Wrong amount position
### Please confirm the following

- [X] I checked [the current issues](https://github.com/Ivy-Apps/ivy-wallet/issues) for duplicate problems

### Describe the bug

When displaying the total amount in homescreen or account view, the number is offsetted to the bottom and not centered relative to the currency label.

### To Reproduce

1. Open the app and/or open account detail page

### Expected behavior

Total amount number should be centered relative to the currency label

### Smartphone

Pixel 8 Pro

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by nkhar**
## Pull Request (PR) Checklist
Please check if your pull request fulfills the following requirements:
- [x] The PR is submitted to the `main` branch.
- [x] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) and my PR doesn't break the rules.
- [x] I've read the [Architecture Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Architecture.md).
- [x] I confirm that I've run the code locally and everything works as expected.
- [ ] 🎬 I've attached a **screen recoding** of the changes. 

> Tip: drag & drop the video to the PR description.

## What's changed?

Describe with a few bullets **what's new:**
- BalanceRow is now a single Text Component instead of having 2 separate Text components for balance and currency

Before | After
--|--
 | 

> 💡 Tip: Please, attach screenshots and screen recordings. It helps a lot!

## Risk Factors

**What may go wrong if we merge your PR?**

- Design maybe different than expected
- Have not checked on very large numbers

**In what cases your code won't work?**

- Not sure

## Does this PR closes any GitHub Issues?

Check **[Ivy Wallet Issues](https://github.com/Ivy-Apps/ivy-wallet/issues)**.

- Closes #3043

### CODE REVIEW

File: temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/components/BalanceRow.kt [hunk 1]
```diff
@@ -112,7 +112,7 @@ fun BalanceRow(
             verticalAlignment = Alignment.CenterVertically,
             horizontalArrangement = Arrangement.SpaceBetween
         ) {
-            if (currencyUpfront || doubleRowDisplay) {
+            if (doubleRowDisplay) {
```
Comment (ILIYANGERMANOV) [inline temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/components/BalanceRow.kt:115]: Can you test and upload screenshots where `doubleRowDisplay` is true and false?

Comment (ILIYANGERMANOV) [reply temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/components/BalanceRow.kt:115]: If `doubleRowDisplay` is never true we can get rid of it

Comment (nkhar) [reply temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/components/BalanceRow.kt:115]: doubleRowDisplay is true for HeaderStickyRow composable.

I will check true and false and upload appropriate screenshots tomorrow.

should I replace line 124 `if (!doubleRowDisplay)` with just an else.

**Comment by ILIYANGERMANOV** [comment]
@nkhar I'll merge the fix as it's a good improvement 💯

**Pull request merged**